trials/survey.js

```diff
@@ -0,0 +1,30 @@
+const { baseStimulus } = require("../lib/markup/stimuli");
+
+/**
+ * @description
+ * Builds a trial with a question and with free response text fields. The subject types in answers.
+ * @module
+ * @param {Object} options
+ * @param {string} options.stimulus - Onscreen stimulus in HTML to be shown in the trial, if not set default text is empty. If the stimulus is not provided, message should be provided as a string. (default: "")
+ * @param {string} options.message - Onscreen message to be shown in the trial. (default: "")
+ */
+
+module.exports = function (options) {
+  const defaults = {
+    stimulus: "",
+    message: "",
+  };
+  const { stimulus, message } = { ...defaults, ...options };
+
+  const stimulusOrMessage =
```

Comment:
does this need the photodiode option?

Reply:
do all trials need that? cc: @nicoleprovenza

Reply:
just asked Nicole this does not require the photodiode box